docs/classic/release-notes.md

```diff
@@ -2,8 +2,12 @@
 id: release-notes
 title: Release Notes
 ---
+### Important Notes
 
-### WebRTC Plan-B Deprecation
+#### Spring '22 Release
+The Salesforce Spring '22 release introduces a change that will likely cause the installation of any version of the adapter before 5.18 to fail. This issue will not affect anyone who uses a version of the adapter before 5.18, only those who want to install an older version.
```

Comment:
Should we say "install or update to an older version"?

Reply:
Yea, that makes sense.